### PR TITLE
Migrated changes related to perfData, mergeMethod and lossMetrics

### PR DESCRIPTION
**Pull request by girisairam450**

### CODE REVIEW

**Comment by suresh-ls** [comment]
Important:  From your branch, Wheel file also has to be uploaded under lib/

File: lib/src/python-client/swarmlearning/client/app2if.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def __init__(
       , checkinModelOnTrainEnd: str
       , nodeWeightage: int
       , trainingContract: str
+      , mergeMethod: str
```
[the remark this answers is not in the document]
Comment (girisairam450) [reply lib/src/python-client/swarmlearning/client/app2if.py:47]: Done

File: lib/src/python-client/swarmlearning/swci.py [hunk 2]
```diff
@@ -524,3 +524,8 @@ def uploadTaskDefintion(self, taskFilePath):
             return r.text
         except Exception as e:
             raise RuntimeError('Task definition file upload failed')            
+
```
[the remark this answers is not in the document]
Comment (girisairam450) [reply lib/src/python-client/swarmlearning/swci.py:529]: Done

File: lib/src/python-client/swarmlearning/tf.py [hunk 3]
```diff
@@ -97,6 +98,9 @@ def __init__(self, syncFrequency, minPeers, trainingContract=DEFAULT_TRAINING_CO
                        from swarmlearning.tf import SwarmCallback                       
                        swCallback = SwarmCallback(syncFrequency=128, minPeers=3)
                        swCallback.logger.setLevel(logging.DEBUG)
+        :param totalEpochs: Total epochs used in local training. 
+                            This is needed to display training progress or ETA of training.
+                            WARNING: "With out this, training progress display might have some limitations."
```
[the remark this answers is not in the document]
Comment (girisairam450) [reply lib/src/python-client/swarmlearning/tf.py:103]: Done

**Review by suresh-ls — APPROVED** [review]

**Review by suresh-ls — COMMENTED** [review]
Attach the test logs of a successful run from this wheel file..

**Pull request merged**